charts/incubator/penpot/templates/_backend.tpl

```diff
@@ -0,0 +1,14 @@
+{{/* Define the backend container */}}
+{{- define "penpot.backend" -}}
+image: {{ .Values.backendImage.repository }}:{{ .Values.backendImage.tag }}
+imagePullPolicy: '{{ .Values.image.pullPolicy }}'
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /opt/data
+
```

Comment:
All `dicts` we have in values.yaml does not apply in any template file.
It's custom things we build around them.
Here only native kubernetes options are available.